Getting-Started/Design/Rendering-Content/index.md

```diff
@@ -98,7 +98,7 @@ You can do this by querying content relative to your current page in template vi
 </ul>
 ```
 
-You can use the Query Builder in the template editor to build more advanced queries
+You can use the Query Builder in the template editor to build more advanced queries:
```

Comment:
```suggestion
You can use the Query Builder in the template editor to build more advanced queries.
```

Reply:
That is looking good to me. I will go ahead and commit changes.